### PR TITLE
Add ability to dump values that is bind by CheBootstrap

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Add ability to dump values that is bind by CheBootstrap

### What issues does this PR fix or reference?
n/a
#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]

**Comment by skabashnyuk** [comment]
ci-test

File: core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java [hunk 1]
```diff
@@ -438,18 +438,22 @@ private void bindProperty(String prefix, String name, String value) {
       String key = prefix == null ? name : (prefix + name);
       Set<String> aliasesForName = bindMap.get(name);
       if (value == null) {
+        LOG.debug("Binding `{}` to `null`", key);
         bind(String.class).annotatedWith(Names.named(key)).toProvider(Providers.<String>of(null));
         if (aliasesForName != null) {
           for (String alias : aliasesForName) {
+            LOG.debug("Binding `{}` to `null`", alias);
```
Comment (metlos) [inline core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java]: Not precise if `prefix != null`

**Review by metlos — APPROVED** [review]
LGTM, apart from the prefixed alias binding.

File: core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java [hunk 2]
```diff
@@ -438,18 +438,22 @@ private void bindProperty(String prefix, String name, String value) {
...
                 .annotatedWith(Names.named(prefix == null ? alias : prefix + alias))
                 .toProvider(Providers.<String>of(null));
           }
         }
       } else {
+        LOG.debug("Binding `{}` to `{}`", key, value);
         bindConstant().annotatedWith(Names.named(key)).to(value);
         if (aliasesForName != null) {
           for (String alias : aliasesForName) {
+            LOG.debug("Binding `{}` to `{}`", key, value);
```
Comment (metlos) [inline core/commons/che-core-commons-inject/src/main/java/org/eclipse/che/inject/CheBootstrap.java]: This is wrong, I think? We're binding the aliases here, not the key itself. Also lookout for the `prefix` value.

**Review by mshaposhnik — APPROVED** [review]

**Comment by skabashnyuk** [comment]
ci-test

**Pull request merged**